app/components/avo/fields/common/files/view_type/grid_item_component.html.erb

```diff
@@ -8,8 +8,8 @@
       <%= video_tag(helpers.main_app.url_for(file), controls: true, preload: false, class: 'w-full') %>
     <% else %>
       <a
-        <% if can_download_file? %>href="<%= helpers.main_app.url_for(file) %>" target="_blank" rel="noopener noreferrer"<% end %>
-        class="relative flex flex-col justify-evenly items-center px-2 rounded-lg border bg-white border-gray-500 min-h-24 <%= 'hover:bg-gray-100 transition' if can_download_file? %>"
+        <% if file.representable? %>href="<%= helpers.main_app.url_for(file) %>" target="_blank" rel="noopener noreferrer"<% end %>
+        class="relative flex flex-col justify-evenly items-center px-2 rounded-lg border bg-white border-gray-500 min-h-24 <%= 'hover:bg-gray-100 transition' if file.representable? %>"
```

Comment:
It's important to check `can_download_file?` to enforce authorization properly. Otherwise, users without download permissions could still open and inspect the PDF.